SConstruct

```diff
@@ -1108,13 +1108,14 @@ if env['system_sundials'] == 'y':
 
     # Ignore the minor version, e.g. 2.4.x -> 2.4
     env['sundials_version'] = '.'.join(sundials_version.split('.')[:2])
-    if env['sundials_version'] not in ('2.4','2.5','2.6','2.7','3.0','3.1','3.2','4.0','4.1','5.0'):
+    system_sundials_version = LooseVersion(env['sundials_version'])
```

Comment:
This isn't necessarily associated with the system SUNDIALS though, so I think just `sundials_ver` would be sufficient and shorter

Reply:
Done